### ISSUE TITLE
Any interest in supporting Postgres RLS (row level security)?

### ISSUE THREAD

**Issue by jtanios** [Finbuckle/Finbuckle.MultiTenant#730]
Adding security at the row level greatly enhances tenant isolation and eliminates the need for query filtering.

I have a working prototype of the solution.

**Comment by AndrewTriesToCode** [comment]
Hi,
I would be interested -- probably as part of a new package. Does it integrate with EFCore? I'm not a Postgres expert but I can see why it would be useful.

**Comment by jtanios** [comment]
Yes, it uses `DbConnectionInterceptor` and `NpgsqlMigrationsSqlGenerator` to implement RLS.

What would be the best way to add this functionality? Create a new package that uses `Finbuckle.Multitenant`?

**Comment by AndrewTriesToCode** [comment]
I'm marking this pinned not inactive because I think it merits more looking into.
